arch/riscv/Kconfig

```diff
@@ -38,6 +38,24 @@ config RISCV_ALWAYS_SWITCH_THROUGH_ECALL
 	  and most people should say n here to minimize context switching
 	  overhead.
 
+config RISCV_ENABLE_FRAME_POINTER
```

Comment:
would be nice to make this a generic arch kconfig (CONFIG_FRAME_POINTER), we now have two architectures doing this